IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory/README.md

```diff
@@ -0,0 +1,113 @@
+# Deploying NGSA Memory Manually
+
+## Prerequisites
+
+- A PnP AKS Deployment with Traefik
+
+## Install NGSA-Memory
+
+- Apply the `ngsa-memory.yaml`
+
+    ```bash
+    kubectl apply -f ./ngsa-memory
+    ```
+- Apply the new traefik configuration
+
+    ```bash
+    # REPO_ROOT=Root directory of this repository
```

Comment:
do we set REPO_ROOT earlier? Should we include instructions for checking / setting?

Reply:
Updated path to remove $REPO_ROOT variable, as it is being set in a different ReadMe. I'll use a relative path instead.